kafka-connect-transforms/README.md

```diff
@@ -24,6 +24,26 @@ The `CopyValue` SMT copies a value from one field to a new field.
 "transforms.copyId.target.field": "id_copy",
 ```
 
+# KafkaMetadataTransform
+_(Experimental)_
+
+The `KafkaMetadata` injects `topic`, `partition`, `offset`, `record timestamp`.
```

Comment:
```suggestion
The `KafkaMetadata` injects `topic`, `partition`, `offset`, `record_timestamp`.
```

Although I'm still pushing for just `timestamp`.

Reply:
Updated the code but forgot this esction of the docs, will fix.